js/main.js

```diff
@@ -1,3 +1,4 @@
-import { getPictures } from './data';
+import { getPictures } from './data.js';
+import { renderThumbnails } from './thumbnails.js';
 
-getPictures();
+renderThumbnails(getPictures());
```

Comment:
```suggestion
const pictures = getPictures();
renderThumbnails(pictures);
```

Так будет чуть проще читаться код